packages/terra-application-navigation/src/extensions/Extension.module.scss

```diff
@@ -9,10 +9,13 @@
     justify-content: center;
     outline: none;
     position: relative;
-
-    &:hover {
-      .extension-inner {
-        color: var(--terra-application-navigation-extension-hover-color, #fff);
+    
+    /* stylelint-disable max-nesting-depth */
```

Comment:
does this need to be re-enabled or can we only disable the next line?

Reply:
Disabled for next line only 0c9bba3